docs/overview.md

```diff
@@ -7,117 +7,101 @@ sidebar_label: Overview
+Local Volumes are accessible only from a single node in the cluster. Pods using Local Volume have to be scheduled on the node where volume is provisioned. Local Volumes are typically preferred for distributed workloads like Cassandra, MongoDB, Elastic, etc that are distributed in nature and have high availability built into them. 
 
-### Local Volumes 
+Depending on the type of storage attached to your Kubernetes worker nodes, you can select from different flavors of Dynamic Local PV - Hostpath, Device, LVM, ZFS or Rawfile.
 
-Local Volumes are accessible only from a single node in the cluster. Pods using Local Volume have to be scheduled on the node where volume is provisioned. Local Volumes are typically preferred for distributed workloads like Cassandra, MongoDB, Elastic, etc that are distributed in nature and have high availability built into them. 
+### Local Volume Quickstart Guides
 
-Depending on the type of storage attached to your Kubernetes worker nodes, you can select from different flavors of Dynamic Local PV - Hostpath, Device, ZFS or Rawfile.
+Installing OpenEBS in your cluster is as simple as running a few `kubectl` or `helm` commands. Here are the list of our Quickstart guides with detailed instructions for each storage engine.
 
-### Replicated Volumes (aka Highly Available Volumes)
+- [Local PV hostpath](/docs/next/uglocalpv-hostpath.html)
+- [Local PV device](/docs/next/uglocalpv-device.html)
+- [ZFS Local PV](https://github.com/openebs/zfs-localpv)
+- [LVM Local PV](https://github.com/openebs/lvm-localpv)
+- [Rawfile Local PV](https://github.com/openebs/rawfile-localpv)
+
+## Replicated Volumes
 
 Replicated Volumes as the name suggests, are those that have their data synchronously replicated to multiple nodes. Volumes can sustain node failures.  The replication also can be setup across availability zones helping applications move across availability zones. 
 
 Replicated Volumes also are capable of enterprise storage features like snapshots, clone, volume expansion and so forth. Replicated Volumes are a preferred choice for Stateful workloads like Percona/MySQL, Jira, GitLab, etc. 
 
 Depending on the type of storage attached to your Kubernetes worker nodes and application performance requirements, you can select from Jiva, cStor or Mayastor. 
 
-### Selecting the right storage engine
+### Replicated Volume Quickstart Guides
+
+Installing OpenEBS in your cluster is as simple as running a few `kubectl` or `helm` commands. Here are the list of our Quickstart guides with detailed instructions for each storage engine.
+
+- [Mayastor](/docs/next/ugmayastor.html)
+- [cStor](https://github.com/openebs/cstor-operators/blob/master/docs/quick.md)
+- [Jiva](https://github.com/openebs/jiva-operator)
+
+## Selecting the right storage engine
```

Comment:
Are we not mentioning backup and restore?

Reply:
yeah.. removed it for now. To avoid confusion.